rust/schema/definable/function.rs

```diff
@@ -20,11 +23,12 @@ pub struct Function {
     span: Option<Span>,
     pub signature: Signature,
     pub block: FunctionBlock,
+    pub unparsed: String,
```

Comment:
Flag a hack.
I'm supposed to use the span for this. But then I have to pass the unparsed query in the core functions.
I'm not sure whether I want to do that today or leave this in as a hack.

Reply:
We're happy with this for now.